tst/EnergyPlus/unit/CMakeLists.txt

```diff
@@ -1,273 +1,274 @@
-if(USE_GLYCOL_CACHING)
-  add_definitions("-DEP_cache_GlycolSpecificHeat")
-endif()
+if (USE_GLYCOL_CACHING)
```

Comment:
Is there a reason this file was reformatted here?  Should've just been a one line addition for the new unit test file...

Reply:
CLion kept forcing this on me when I was updating this file for the one line addition.  If it's an issue, I can try to revert this back and do this manually to not effect the formatting.